src/visualization/StaticGraph/Makefile

```
@@ -1,6 +1,6 @@
 include ../../common.mk
 CC = g++
-OBJ = main.o Graph.o Node.o GraphRenderer.o Edge.o State.o colormap.o colorlist.o ArrowShape.o FuncArrowShape.o DrawNode.o
+OBJ = main.o Graph.o Node.o GraphRenderer.o Edge.o State.o colormap.o colorlist.o ArrowShape.o FuncArrowShape.o DrawNode.o Button.o
```

```suggestion
OBJ = main.o Graph.o Node.o GraphRenderer.o Edge.o State.o colormap.o colorlist.o ArrowShape.o FuncArrowShape.o \
    DrawNode.o Button.o
```
replace empty space after line-break with TAB